clusters/app.ci/grafana-loki/admin_grafana.yaml

```diff
@@ -0,0 +1,109 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: grafana-loki
+---
+kind: Group
+apiVersion: v1
+metadata:
+  name: grafana-loki-admins
+users:
+- vrutkovs
+- periklis
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: grafana-loki-admin
+  namespace: grafana-loki
+subjects:
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: grafana-loki-admins
+roleRef:
+  kind: Role
+  apiGroup: rbac.authorization.k8s.io
+  name: admin
```

Comment:
`view` ? You should look to change YAML in the cluster using `git`

Reply:
Right, updated too view